src/Code.js

```diff
@@ -58,27 +38,47 @@ function setApiKey() {
   const button = result.getSelectedButton();
   const text = result.getResponseText();
   if (button === ui.Button.OK) {
-    setProperty(PROP_MB_API_KEY, text);
-    ui.alert(`API key is ${text}.`);
+    try {
+      setProperty(PROP_MB_API_KEY, text);
+      ui.alert(`API key is ${text}`);
+    } catch (e) {
+      ui.alert(e.message);
+    }
```

Comment:
I know this was also done in a separate PR, but is there a way to set the Deployment ID and API in a single popup, as a convenience? Or not available in Google Sheets?

Reply:
To be honest, it was my lazy I will check this out like a form  🔍

Reply:
Can I create a follow-up ticket for this? because I am testing custom dialog with html but:

- [Need to add another permission scope](https://developers.google.com/apps-script/reference/base/ui#showModalDialog(Object,String)): `https://www.googleapis.com/auth/script.container.ui` it seems not related to our add-on features.
![image](https://user-images.githubusercontent.com/16159767/90375982-b7c18380-e0b0-11ea-8019-b4bdb6955966.png) 
- Create custom CSS to keep style consistency or make all dialogs with custom dialogs >.<;;

What do you think?

Reply:
Yes, let's do as a follow-on!

Reply:
Follow-up ticket: https://github.com/curvegrid/multibaas-for-google-sheets/issues/27